### PR TITLE
Add DAC support for the STM32L4

### PR DESCRIPTION
**Pull request by ervanalb**
Tested on STM32 NUCLEO-L432 board

### CODE REVIEW

File: include/libopencm3/stm32/l4/memorymap.h [hunk 1]
```diff
@@ -62,6 +62,7 @@
 #define USB_PMA_BASE			(PERIPH_BASE_APB1 + 0x6c00)
 #define POWER_CONTROL_BASE		(PERIPH_BASE_APB1 + 0x7000)
 #define DAC1_BASE			(PERIPH_BASE_APB1 + 0x7400)
+#define DAC_BASE			DAC1_BASE
```
Comment (ervanalb) [inline include/libopencm3/stm32/l4/memorymap.h]: I could have just renamed `DAC1_BASE` but I did it this way so as to not break anybody's code that is relying on `DAC1_BASE` existing. Happy to switch it back if you prefer.

Comment (karlp) [reply include/libopencm3/stm32/l4/memorymap.h]: We've done similar things on F0 that only has one dac as well.  If l4 only has the one dac, then this is perfectly fine here too.

Comment (karlp) [reply include/libopencm3/stm32/l4/memorymap.h]: Though, if there was no user of DAC before, can we just always have DAC1?

Comment (karlp) [reply include/libopencm3/stm32/l4/memorymap.h]: Actually, looking at the rest of the library, I think DAC1 should be dropped.  until/if they really do bring out second dac.  Everyone else uses DAC_BASE.

**Comment by karlp** [comment]
I've merged this without the DAC1_BASE, as mentioned above.  thanks for this!

**Pull request closed without merging**